src/compute_order_functions.py

```
@@ -0,0 +1,207 @@
+        B_ij:   1 when c_i is the j-th vertex in the mesh
+
+    Args:
+        vertices ((v, 3), float): the vertices of the mesh
+        condition_edges ((n, 2), int): the edges of the path condition
+        path ((n, 3), float): the points on the mesh that should receive the value 0 in the computed function
+
+    Returns:
+        ((n, v), float) a condition matrix that makes sure that g(path)=0.
+    """
+    if np.any(condition_edges == -1):
+        raise ValueError("Received points that are not on a mesh edge or vertex")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)